### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Abestanis**
*TLDR*: This updates the dependencies and changes how `content.scss` is processed.

Hey, it has been a while, but I want to start looking into integrating KeePassNatMsg.
I'm on a new development machine and I have a newer version of nodejs installed (`v18.10.0`). I'm not sure if that is the reason, but I got errors when building the code, and once I got it to build all the styles from `content.scss` were not properly loaded.

So I decided to try and update the dependencies to the latest version. It took me a while messing with the `webpack.config.js`, but I finally managed to come up with a version that works. From my tests the functionality of the extension is not affected by this, everything still works as before.

### CODE REVIEW

**Comment by RoelVB** [comment]
There is no `content.scss` anymore in CKP v2 and a lot of dependencies will change.
